src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -688,6 +699,59 @@ class DevTask extends AbstractFeatureTask {
             }
         }
 
+        private boolean hasInstallationConfigChanged(Project newProject, Project oldProject) {
+            // The following project configuration is used for the Liberty installation
+            // liberty extension: installDir, baseDir, cacheDir, outputDir, userDir, runtime
+            // libertyRuntime dependency
+            // liberty->install block: type, runtimeUrl, version, useOpenLiberty
+            boolean foundChange = false
+            if (newProject.liberty.installDir != oldProject.liberty.installDir) {
+                logger.debug('liberty.installDir changed')
+                foundChange = true
+            }
+            if (newProject.liberty.baseDir != oldProject.liberty.baseDir) {
```

Comment:
A similar method on the LMP side used an `else-if` pattern and returned `true` as soon as a change was found. Should we follow that strategy here or do we want to make sure to debug every line that was changed?

Reply:
Right...I was trying to give as much feedback as possible if multiple things changed. But I could switch it to just return true on the first change detected. I don't have a problem with that. Let me know if you want me to change it.

Reply:
I changed it. Didn't see the point in an `else-if` pattern, but I did `return true` as soon as an `if` evaluates to `true`.